common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -989,3 +989,52 @@ sBssidInfo:
   downlink_estimated_mac_data_rate_mbps: uint32_t
   uplink_estimated_mac_data_rate_mbps: uint32_t
   sta_measured_uplink_rssi_dbm_enc: uint8_t
+
+sClientConfig:
+  _type: struct
+  stay_on_initial_radio:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  stay_on_selected_device:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  selected_bands:
+    _type: char
+    _length: ["beerocks::message::SELECTED_BANDS_LENGTH"]
+  time_life_delay_days:
+    _type: uint32_t
+    _comment: optional parameter, Timelife delay of the client configuration (if configured), otherwise 0.
+
+sClient:
+  _type: struct
+  sta_mac:
+    _type: sMacAddr
+    _comment: Client MAC
+  timestamp_sec:
+    _type: uint32_t
+    _comment: Time of last client update (in Seconds)
+  stay_on_initial_radio:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  stay_on_selected_device:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  selected_bands:
+    _type: char
+    _length: ["beerocks::message::SELECTED_BANDS_LENGTH"]
+    _comment: comma-separated selected bands [2.4,5H,5L,6]
+  support_11k:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  support_11v:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  support_steering:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
+  dual_band:
+    _type: int8_t
+    _comment: 1 for true, 0 for false, -1 for "not configured".
```

Comment:
same here see my comment on the BML_CLIENT structure

Reply:
Fixed